src/inference/CMakeLists.txt

```diff
@@ -138,7 +134,7 @@ ie_faster_build(${TARGET_NAME}_obj
 )
 
 target_compile_definitions(${TARGET_NAME}_obj PRIVATE
-    IMPLEMENT_INFERENCE_ENGINE_API
+    IMPLEMENT_OPENVINO_API
```

Comment:
I think we need completely drop `IMPLEMENT_INFERENCE_ENGINE_API` then from OpenVINO code

Reply:
> IMPLEMENT_INFERENCE_ENGINE_API

@ilya-lavrenov I just found IMPLEMENT_INFERENCE_ENGINE_API in common.hpp, ie_api.h and CMakeLists.txt, any other places should be fixed?

Reply:
I suppose `grep` knows better than me all the places where `IMPLEMENT_INFERENCE_ENGINE_API` should be removed :)

Reply:
> I suppose `grep` knows better than me all the places where `IMPLEMENT_INFERENCE_ENGINE_API` should be removed :)

<img width="567" alt="grep_IMPLEMENT_INFERENCE_ENGINE_API" src="https://user-images.githubusercontent.com/17422402/223034367-222a8b99-df01-481b-a70a-7ffde32e95f5.png">
It seems that there are 3 places in openvino code. @ilya-lavrenov

Reply:
yes